extensions/sql-database-projects/package.json

```diff
@@ -388,6 +397,11 @@
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.file.externalStreamingJob",
           "group": "5_dbProjects_streamingJob"
         },
+        {
+          "command": "sqlDatabaseProjects.openInDesigner",
+          "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.file.table",
```

Comment:
We might also want to check `config.workbench.enablePreviewFeatures`

Ref: https://github.com/microsoft/azuredatastudio/blob/727f37d6a5df7b504800861dcd263cbe8bd0707a/extensions/mssql/package.json#L509